portfolio/src/main/webapp/index.html

```diff
@@ -5,25 +5,58 @@
     <title>Jazmine's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    
+    <style type = "text/css">
+    #smile {
+        width: 300px;
+        height: auto;
+    }
+    #picture1{
+        width: auto;
+        height: 250px;
+    }
+    #picture2{
+        width: auto;
+        height: 250px;
+    }
+    #picture3{
+        width: 200px;
+    }
+    #picture4{
+        width: 200px;
+    }
+    #picture5{
+        width: 200px;
+    }
+    #book{
+        width: auto;
+        height: 200px;
+    }
+    #tvshow{
+        width: auto;
+        height: 200px;
+    }
+    </style>
+
   </head>
   <body>
     <div id="content">
       <h1>Jazmine's Portfolio</h1>
       <p>Hello! My name is Jazmine Bullock.</p>
-      <img width = "340" src="images/pic-1.jpg" alt = "hey!"/>
+      <p><img src="images/pic-1.jpg" alt = "hey!" id = "smile"></p>
           <p>I am a Computer Science student at North Carolina A&T University.</p>
           <p>I also am in the Blue and Gold marching machine! Go piccolos!</p>
-          <img width = "300" src="images/NCAT.png" alt = "school"/>
-          <img width = "300" src="images/group-pic.jpg" alt = "piccolos!"/>
+          <p><img src="images/NCAT.png" alt = "school" id = "picture1"></p>
```

Comment:
Just a suggestion for the future - The IDs may be easier to reason about (especially if the selectors are in a different file) if they describe the element rather than the order of the elements "school" vs "picture1".